cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -1095,6 +1144,10 @@ func generateMountsModeRecover(mst *initramfsMountsState) error {
 		return err
 	}
 
+	if err := generateMountsCommonInstallRecoverContinue(mst, model, snaps); err != nil {
```

Comment:
nitpick: keeping `generateMountsCommonInstallRecover` as a wrapper that calls two functions (with Start and Continue postfixes) so `generateMountsModeRecover` and `generateMountsModeFactoryReset` do not need to change would be nice.

Reply:
Done in 29793ee15235e0474adf1394253608f232234a3d